src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,25 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
+import java.util.ArrayList;
+import java.util.List;
+
 public class Bulldozer extends Machine {
+    private static final int COUNT_BULLDOZERS = 3;
+
     public Bulldozer() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
```

Comment:
task description:
You should have at least 3 implementations: BulldozerProducer, ExcavatorProducer, TruckProducer

you should create 3 separate classes: BulldozerProducer, ExcavatorProducer, TruckProducer that extend MachineProducer
and move this logic to these classes